explorer/app/components/Project/common/projectTransformer.ts

```diff
@@ -2,6 +2,19 @@
 import { ProjectResponse } from "app/models/responses";
 import { Contact } from "../components/Contacts/contacts";
 
+/**
+ * Builds project citation path from projectId.
+ * @param project - Project response model return from API.
+ * @returns string representation of project citation path.
+ */
+export function buildProjectCitationPath(project?: ProjectResponse): string {
+  if (!project) {
+    return "";
+  }
+
+  return `explore/projects/${project.projects[0].projectId}`;
```

Comment:
`explore/projects/` is available in the `SiteConfig` (see `redirectRootToPath`). Let me know if you would like this to be used instead - we can access the value `redirectRootToPath` via the hook `useConfig()`. See line `19` in `Citation` component.